ca/django_ca/tests/tests_constants.py

```diff
@@ -26,6 +26,6 @@ class ReasonFlagsTestCase(TestCase):
     def test_completeness(self):
         """Test that our list completely mirrors the cryptography list."""
         self.assertEqual(
-            list(sorted([(k, v.value) for k, v in ReasonFlags.__members__.items()])),
-            list(sorted([(k, v.value) for k, v in x509.ReasonFlags.__members__.items()]))
+            list(sorted([(k, v.value) for k, v in ReasonFlags.__members__.items()])),  # pylint: disable=E1101
+            list(sorted([(k, v.value) for k, v in x509.ReasonFlags.__members__.items()]))  # pylint: disable=E1101
```

Comment:
CI croaked with
```
ca/django_ca/tests/tests_constants.py:29:50: E1101: Function '__members__' has no 'items' member (no-member)
ca/django_ca/tests/tests_constants.py:30:50: E1101: Function '__members__' has no 'items' member (no-member)
```

Those `pylint` pragmas improved the situation, see also [1].

[1] https://stackoverflow.com/questions/35990313/avoid-pylint-warning-e1101-instance-of-has-no-member-for-class-with-dyn